5.0/fr/0x05-For-Users-Of-4.0.md

```diff
@@ -0,0 +1,90 @@
+
+## Introduction
+
+Les utilisateurs familiarisés avec la version 4.x de la norme trouveront utile de consulter les principales modifications apportées à la version 5.0, notamment les mises à jour de contenu, de portée et de philosophie sous-jacente.
+
+Sur les 286 exigences de la version 4.0.3, seules 11 restent inchangées, tandis que 15 ont subi des ajustements grammaticaux mineurs sans altérer leur sens. Au total, 109 exigences (38 %) ne sont plus des exigences distinctes dans la version 5.0 : 50 ont été simplement supprimées, 28 ont été supprimées car doublons et 31 ont été fusionnées avec d'autres exigences. Les autres ont été révisées. Même les exigences n'ayant pas subi de modifications substantielles ont des identifiants différents à la suite d'une réorganisation ou une restructuration.
+
+Pour faciliter l'adoption de la version 5.0, des documents de correspondance sont fournis pour aider les utilisateurs à identifier les correspondances entre les exigences de la version 4.x et celles de la version 5.0. Ces correspondances ne sont pas liées au numéro de version et peuvent être mises à jour ou clarifiées si nécessaire.
+
+## Philosophie des exigences
+
+### Portée et orientation
+
+La version 4.x incluait des exigences qui ne correspondaient pas au périmètre prévu de la norme ; celles-ci ont été supprimées. Les exigences qui ne répondaient pas aux critères du périmètre de la version 5.0 ou qui n'étaient pas vérifiables ont également été exclues.
+
+### Priorité aux objectifs de sécurité plutôt qu'aux mécanismes
+
+Dans la version 4.x, de nombreuses exigences se concentraient sur des mécanismes spécifiques plutôt que sur les objectifs de sécurité sous-jacents. Dans la version 5.0, les exigences sont centrées sur les objectifs de sécurité, ne référençant des mécanismes particuliers que lorsqu'ils constituent la seule solution pratique, ou les fournissant à titre d'exemple ou de guide complémentaire.
+
+Cette approche reconnaît que plusieurs méthodes peuvent exister pour atteindre un objectif de sécurité donné et évite toute prescription inutile susceptible de limiter la flexibilité organisationnelle.
+
+De plus, les exigences traitant d'une même préoccupation de sécurité ont été consolidées lorsque cela était nécessaire.
+
+### Décisions de sécurité documentées
+
+Bien que le concept de décisions de sécurité documentées puisse paraître nouveau dans la version 5.0, il s'agit d'une évolution des exigences antérieures liées à l'application des politiques et à la modélisation des menaces dans la version 4.0. Auparavant, certaines exigences exigeaient implicitement une analyse pour éclairer la mise en œuvre des contrôles de sécurité, comme la détermination des connexions réseau autorisées.
+
+Afin de garantir la disponibilité des informations nécessaires à la mise en œuvre et à la vérification, ces attentes sont désormais explicitement définies comme des exigences de documentation, ce qui les rend claires, exploitables et vérifiables.
```

Comment:
"exploitables" was "actionnable" in english. I think "actionable" is fine in French in this context (more precise) ?